integrations/axum/Cargo.toml

```diff
@@ -15,7 +15,7 @@ categories = ["network-programming", "asynchronous"]
 async-graphql = { path = "../..", version = "3.0.37" }
 
 async-trait = "0.1.51"
-axum = { version = "0.4", features = ["ws", "headers"] }
+axum = { version = "0.5.1", features = ["ws", "headers"] }
```

Comment:
Why 0.5.1 instead of 0.5?

Reply:
It's the latest version, but no other reason other than my project needing  a version >= `0.5`.
If there's a reason to use `0.5` over `0.5.1`, this pull request's fixes still apply - assuming a `0.5` in the `Cargo.toml`.

Though, the release notes lead me to think they're almost the same: https://github.com/tokio-rs/axum/releases/tag/axum-v0.5.1

Reply:
Sorry, I misunderstood how Cargo interprets version requirements. This seems fine

Reply:
No worries 👍🏼